leakcanary-android-core/src/main/java/leakcanary/internal/NotificationReceiver.kt

```diff
@@ -40,7 +40,7 @@ internal class NotificationReceiver : BroadcastReceiver() {
     ): PendingIntent {
       val broadcastIntent = Intent(context, NotificationReceiver::class.java)
       broadcastIntent.action = action.name
-      val flags = if (Build.VERSION.SDK_INT >= 23) {
+      val flags = if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
```

Comment:
Most of the rest of the codebase uses direct numbers.

Reply:
https://twitter.com/JakeWharton/status/1049421501559128065

Reply:
No problem, thanks for pointing it out.